harvester/create.go

```diff
@@ -0,0 +1,202 @@
+package harvester
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/harvester/go-harvester/pkg/builder"
+	goharverrors "github.com/harvester/go-harvester/pkg/errors"
+	"github.com/rancher/machine/libmachine/drivers"
+	"github.com/rancher/machine/libmachine/log"
+	"github.com/rancher/machine/libmachine/mcnutils"
+	"github.com/rancher/machine/libmachine/ssh"
+	"github.com/rancher/machine/libmachine/state"
+)
+
+const (
+	keypairNamespace = "harvester-system"
```

Comment:
notes: will need to change keypair to use user-namespace to be compatible with future RBAC.